railties/lib/rails/application/default_middleware_stack.rb

```diff
@@ -31,11 +31,12 @@ def build_stack
           middleware.use ::Rack::MethodOverride
           middleware.use ::ActionDispatch::RequestId
 
+          # Must come before Rails::Rack::Logger to properly log request_id
```

Comment:
so if I read the comment correctly, could you comment the ``Logger` instead with:
```
# Must come after ActionDispatch::RemoteIP to properly log ip address
```